### PR TITLE
get bracketarrow working again

### PR DESCRIPTION
**Pull request by aklt**
Associated Issue: #3109 

### Summary of Changes

* Revive the small arrow pointing to the variable that the preview popup shows 

### Test Plan

Start debugging a file and mouse over a variable while stepping through code.  The poup
has a small arrow pointing to the variable.

### CODE REVIEW

File: src/components/shared/BracketArrow.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+
+    return dom.div(
+      {
+        className: classNames("bracket-arrow", orientation || "up"),
+        style: { left, top, bottom }
+      },
+      ""
+    );
+  }
+}
```
Comment (jasonLaster) [inline src/components/shared/BracketArrow.js]: oh interesting, why does this need to be a component over a function?

File: src/components/shared/BracketArrow.js [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
 
 BracketArrow.displayName = "BracketArrow";
 
+BracketArrow.propTypes = {
+  orientation: PropTypes.string,
+  left: PropTypes.number,
+  top: PropTypes.number,
+  bottom: PropTypes.number
+};
```
Comment (jasonLaster) [inline src/components/shared/BracketArrow.js]: We're now using props, which is flow based.

Checkout https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/Footer.js#L25-L35

**Comment by aklt** [comment]
@jasonLaster you were right, no reason to use a class when a function suffices.

I tested in firefox 54.0 and changed the position of the arrow which was a bit off.

**Comment by jasonLaster** [comment]
Great. Thanks for the fix

File: src/components/shared/BracketArrow.js [hunk 3]
```diff
@@ -5,12 +5,17 @@ import classNames from "classnames";
...
+  left,
+  top,
+  bottom
+}: {
   orientation: string,
   left: number,
   top: number,
   bottom: number
-) => {
+}) => {
```
Comment (jasonLaster) [inline src/components/shared/BracketArrow.js]: 👍

**Pull request merged**